### PR TITLE
Make qvm-copy/move[-to-vm] one script

### PR DESCRIPTION
**Pull request by marmarta**
Instead of two scripts calling one another, now it's
only one script, with added variable names inside for clarity.
Also hopefully finally fixes the help messages for good.

fixes QubesOS/qubes-issues#4058

### CODE REVIEW

File: qubes-rpc/qvm-copy [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 set -e
 #
-# The Qubes OS Project, https://www.qubes-os.org
+# The Qubes OS Project, http://www.qubes-os.org
```
Comment (marmarek) [inline qubes-rpc/qvm-copy]: This and the line below looks like a downgrade...

File: qubes-rpc/qvm-copy [hunk 2]
```diff
@@ -21,5 +21,68 @@ set -e
...
+	shift
+else
+	export PROGRESS_TYPE=console
+fi
+
+if [ "$TARGET_TYPE" = "vm" ]; then
+  VM="$1"
+  shift
+else
+  VM="\$default"
```
Comment (marmarek) [inline qubes-rpc/qvm-copy]: While at it, please change to the new syntax: `@default`

File: qubes-rpc/qvm-copy [hunk 3]
```diff
@@ -21,5 +21,68 @@ set -e
...
+fi
+
+if [ $PROGRESS_TYPE = console ] ; then
+    FILECOPY_TOTAL_SIZE=$(du --apparent-size -c -- "$@" 2> /dev/null | tail -1 | cut -f 1)
+    export FILECOPY_TOTAL_SIZE
+fi
+
+if [ "$VM" != \$default ]; then
```
Comment (marmarek) [inline qubes-rpc/qvm-copy]: This should be `"$TARGET_TYPE" = "vm"`

**Comment by marmarta** [comment]
pushed an amend with aforementioned issues fixed

**Pull request merged**